larray/viewer/view.py

```diff
@@ -1316,8 +1316,9 @@ def setup_menu_bar(self):
                                           triggered=self.close))
 
         help_menu = menu_bar.addMenu('&Help')
-        help_menu.addAction(create_action(self, _('Online documentation'), shortcut="Ctrl+H",
-                                          triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('Online &Documentation'), shortcut="Ctrl+H", triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('&Getting Started'), triggered=self.open_tutorial))
```

Comment:
IMO, Getting Started is the equivalent of 10 minutes to Pandas which does not correspond to the "tutorial". As I see it, the getting started should be read in maximum 10 minutes and just present the main features of LArray